numba/np/arraymath.py

```diff
@@ -3895,6 +3895,12 @@ def impl(a, dtype=None):
             for i, v in enumerate(a):
                 ret[i] = v
             return ret
+    elif isinstance(a, types.Literal):
```

Comment:
I think it would be a good idea to be specific here, this is for instances of `types.StringLiteral`s only?